tests/functional/external_mode/test_virtualization_storageclass.py

```
@@ -86,52 +83,74 @@ def setup_and_teardown_cnv(self):
-            if ns_handler.is_exist(resource_name=vm_namespace):
-                ns_handler.delete(resource_name=vm_namespace)
-                ns_handler.wait_for_delete(resource_name=vm_namespace, timeout=300)
+        self.sc_handler.delete(resource_name=self.virt_sc_name)
+        sampler = TimeoutSampler(
+            timeout=180,
+            sleep=15,
+            func=self.sc_handler.is_exist,
+            resource_name=self.virt_sc_name,
+        )
+        assert sampler.wait_for_func_status(
+            True
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)